### DISCUSSION TITLE
Add social icons for mastodon & bluesky?

### DISCUSSION THREAD

**Discussion by cemerick** [facebook/docusaurus#10762] [Ideas]
Would a PR adding these be of interest? I think these two probably dodge [the objection re: wordpress](https://github.com/facebook/docusaurus/discussions/10561).

**Comment by slorber** [comment]
I actually think we should add those, and it's not the same as the Wordpress case since those are social platforms, while Wordpress is more or less an implementation detail of a "website"

I can say "I have a bluesky account", but I would just say "I have a website", not "I have a Wordpress website". Who cares Wordpress is the tech running it behind the scene. Similarly we don't add a Docusaurus icon for those that want to link to a Docusaurus site.

**Reply by ilg-ul** [reply]
> but I would just say "I have a website", not "I have a Wordpress website". 

Actually one would say "I have a Wordpress blog"; Wordpress is more than just an implementation detail of a "website".

**Reply by slorber** [reply]
> Actually one would say "I have a Wordpress blog"

Maybe, but it's only because you are the admin that you care. Visitors don't.

As a visitor, what's the purpose of knowing that the link you click on is using WordPress? I just want to visit the website. I don't care if it's using WordPress, Wix, Framer, Joomla, Drupal, Nginx, Apache or whatever.

As a visitor, I care about Bluesky, Instagram, Mastodon etc, because I may use these platforms and want to connect with the blog author there.

**Comment by cemerick** [comment]
> it's not the same as the Wordpress case since those are social platforms

Yeah, I thoroughly agree, was just tying in the only relevant discussion I'd seen.

I'll put in a PR shortly and link here.

**Comment by cemerick** [comment]
PR open now @ https://github.com/facebook/docusaurus/pull/10764
